### ISSUE TITLE
node-lint spurious warnings

### ISSUE THREAD

**Issue by mcandre** [jpolo/node-lint#2]
I'm running node-lint on a Hello World app based on the official NodeJS Hello World example.

```
$ cat hello.js 
#!/usr/bin/env node

var http = require("http");

http.createServer(function (req, res) {
    res.writeHead(200, {"Content-Type": "text/plain"});
    res.end("Hello World\n");
}).listen(8124, "127.0.0.1");

console.log("Server running at http://127.0.0.1:8124/");
$ node-lint hello.js 
♢ /Users/andrew/Desktop/src/mcandre/node/hello.js
  1) var http = require("http");  //line 3
    » 'require' is not defined.
  2) console.log("Server running at http://127.0.0.1:8124/");  //line 10
    » 'console' is not defined.

✗ Invalid » 1 file ∙ 2 errors
```

require and console are defined in the core NodeJS language, no?

**Comment by jpolo** [comment]
The program can also validate browser code or any ecma script. Please try to add /_jslint nodejs: true_/ at the beginning of your file (just after !/usr/bin/env node)
